openmdao/docs/feature_reference/core_features/working_with_derivatives/specifying_partials.rst

```diff
@@ -43,5 +43,4 @@ Here are several examples of how you can specify derivative values for different
     openmdao.jacobians.tests.test_jacobian_features.SimpleCompConst
 
 
-.. tags:: Partial Derivatives
-
+.. tags:: PartialDerivatives
```

Comment:
currently, tags don't work with spaces in them, if it's important that they do, could put in a story.